sunpy/timeseries/timeseriesbase.py

```diff
@@ -119,6 +119,16 @@ def __init__(self, data, meta=None, units=None, **kwargs):
 
 # #### Attribute definitions #### #
 
+    @property
+    def data(self):
+        warnings.warn("Using .data to access the dataframe is discouraged;"
```

Comment:
```suggestion
        warnings.warn("Using .data to access the dataframe is discouraged; "
```
Sorry to bother, but apart from this space all looks 👍 👍 👍